### PR TITLE
OCPBUGS#13791: Updating Nutanix AOS and Prism Central versions

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.13+

Issue:
This PR addresses [ocpbugs-13791](https://issues.redhat.com/browse/OCPBUGS-13791).

Link to docs preview:

[Nutanix version requirements](https://65041--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_nutanix/preparing-to-install-on-nutanix#installation-nutanix-infrastructure_preparing-to-install-on-nutanix)

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Comment by mjpytlak** [comment]
@sgaoshang Ready for QE review. PTAL.

File: modules/installation-nutanix-infrastructure.adoc [hunk 1]
```diff
@@ -13,6 +13,8 @@ You must install the {product-title} cluster to a Nutanix environment that meets
 [cols=2, options="header"]
 |===
 |Component |Required version
-|Nutanix AOS | 5.20.4+ or 6.5.1+
-|Prism Central | 2022.4+
+|Nutanix AOS | Long Term Support (LTS) release 6.5.2.7 or later
+
```
Comment (racedo) [inline modules/installation-nutanix-infrastructure.adoc:17]: We will support 6.5.2.7 or later, regardless LTS or STS. We should remove all references to LTS or STS and just leave it as "Nutanix AOS release 6.5.2.7 or later", which includes both, and both are supported by Nutanix.

**Comment by mjpytlak** [comment]
@sgaoshang Updated versions per @racedo. Please take one more look. Thank you.

**Comment by mjpytlak** [comment]
Got approval from Nutanix (Yannick Struyf) in Slack.

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**